architecture-decision-records/0011-base25-encoding-for-consignment-reference.md

```diff
@@ -0,0 +1,88 @@
+# base25 encoding for consignment references
+
+**Date:** 2020-11-13
+
+## Context
```

Comment:
Could you also mention the things that we considered including in the reference, and why we decided against them? It's the kind of thing we'll forget in a year's time.

I think we talked about:

* Series ID: this was proposed because a consignment currently belongs to a single series, so it might be a useful reference to users. We decided against it because the long-term plan is to remove the concept of series from TDR.
* Transferring body code or name: proposed for the same reason as a series. I'm actually not sure why we decided against this one. Was it because departments can change (e.g. name changes, merges and splits) so it's not as stable an identifier as it would seem?